lib/features/auto-resize/AutoResize.js

```diff
@@ -218,11 +221,9 @@ AutoResize.prototype.getOffset = function(shape) {
  * Get the activation threshold for each side for which
  * resize triggers.
  *
- * @param {djs.model.Shape} shape
- *
  * @return {Object} {top, bottom, left, right}
  */
-AutoResize.prototype.getPadding = function(shape) {
```

Comment:
Please revert this. We do actually call `#getPadding` with the desired argument. This method is meant to be overridden by sub-classes that may apply more specific behavior.

Reply:
Ooops. 🙃